### DISCUSSION TITLE
Need help on homepage Top navigation bar CSS

### DISCUSSION THREAD

**Discussion by chloe8225** [community/community#129545] [Programming Help]
Hello, I am a beginner in coding and trying to edit my homepage section (I use Cargo for my website).
I have the homepage and other work pages, and I only want to show my top navigation bar on other pages and hide in my homepage.
I have tried several ways but it's not changing at all..Please help!

This is HTML in my top navigation"
<img width="418" alt="HTML" src="https://github.com/community/community/assets/173639580/910e1f29-5d19-4a59-8f57-8b2ea05927da">

This will how it show in page
<img width="1181" alt="HTML2" src="https://github.com/community/community/assets/173639580/edfefc79-6375-46bd-b133-551ac10733a6">

I know I should have an id or class to give change but not sure where to put it and how to write CSS to hide it only in the specific page.
I tried giving the id to column set and hide with CSS display: hidden, but this hides my top nav bar in all the pages. Thank you!

**Comment by SaadDev-FR** [comment]
first of all I do not know which framework are you using I mean react,angular,razor etc.
By the way use this 
/* Assuming the navigation bar has an ID of #navbar */
.homepage #navbar {
    display: none;
}

and javascript script add in your project
<script> document.addEventLister("DOMCONTENTLOADED,",function()
{
if(window.location.pathname === "/")
{
document.body.ClassList.add('homepage');

}});
</script>
